### PR TITLE
Creating products

### PR DESCRIPTION
**Pull request by justinchoi001**
Commerce creating products

### CODE REVIEW

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-grouped-product.md [hunk 1]
```diff
@@ -58,12 +58,20 @@ Once you have applied product options to the product, SKUs have to be generated.
...
+1. Click the _3-dot_ icon then _Edit_ next to _default_. (At this point, you can rename the SKU to something more descriptive like _Brakes Set Combo Kit_ on the _Details_ menu.)
 
-1. Click the _3-dot_ icon then _Edit_ next to _default_. (At this point, you can rename the SKU to something more descriptive like _Dog Grooming Kit_ on the _Details_ menu.)
+    ![Add the Price](./creating-a-grouped-product/images/07.png)
+
+1. Click _Publish_.
+
+### Liferay Commerce 2.0 and Below
+
+To set product prices and quantities for a Grouped product in previous versions of Liferay Commerce 2.0:
```
Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-grouped-product.md:69]: > To set product prices and quantities for a Grouped product in previous versions of Liferay Commerce 2.0:

It's redundant to repeat "previous versions..." when the header already says that. Please remove.

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-simple-product.md [hunk 2]
```diff
@@ -32,7 +32,20 @@ After the SKUs have been generated, set the base price and quantities in the inv
...
+    * **Cost**: 0.00
+
+    ![Editing the SKU's price](./creating-a-simple-product/images/03.png)
+
+1. Click _Publish_.
+1. Close the configuration
+
+### Liferay Commerce 2.0 and Below
+
+To set product prices and quantities in previous versions of Liferay Commerce:
```
Comment (jrhoun) [inline docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-simple-product.md:48]: It's redundant to repeat "previous versions..." when the header already says that. Please remove.

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-grouped-product.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-grouped-product.md:69]: Revised and pushed

File: docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-simple-product.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/product-types/creating-a-simple-product.md:48]: Revised and pushed

**Comment by jrhoun** [comment]
Merged. See my changes here: https://github.com/liferay/liferay-learn/commit/4843b5733ca8bb6144b3c48797ae5b5b706e9cca

Thanks!

**Pull request closed without merging**